pkg/asset/installconfig/gcp/validation.go

```diff
@@ -462,6 +465,26 @@ func validateNetworks(client API, ic *types.InstallConfig, fieldPath *field.Path
 	return allErrs
 }
 
+func validateServiceEndpoints(_ API, ic *types.InstallConfig, fieldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	// attempt to resolve all the custom (overridden) endpoints. If any are not reachable,
+	// then the installation should fail not skip the endpoint use.
+	for id, serviceEndpoint := range ic.GCP.ServiceEndpoints {
+		// Ignore e2e.local for unit tests.
+		if serviceEndpoint.URL == "e2e.local" {
+			logrus.Warnf("Skipping service endpoint validation for %s", serviceEndpoint.Name)
```

Comment:
nit: I guess we won't see anyone using the `e2e.local` url. But somehow if it happens, should we include a reason why we skip validating (i.e. `e2e.local URL is used for unit tests. Skipping ...`)?

Reply:
we don't do that usually. We will warn saying skipping. You should never see this as it cannot be valid. This is the same thing we do for AWS.

Reply:
Sounds good! Thanks!!